examples/text-generation/text-generation-pipeline/README.md

```diff
@@ -133,37 +133,17 @@ python ../../gaudi_spawn.py --use_deepspeed --world_size 8 run_pipeline.py \
 
 ### Usage with LangChain
 
-The text-generation pipeline can be fed as input to LangChain classes via the `use_with_langchain` constructor argument. Here is a sample snippet that shows how the pipeline class can be used with LangChain.
-```python
-from langchain.llms import HuggingFacePipeline
-from langchain.prompts import PromptTemplate
-from langchain.chains import LLMChain
-
-# Initialize the pipeline
-pipe = GaudiTextGenerationPipeline(args, logger, use_with_langchain=True)
-
-# Create LangChain object
-llm = HuggingFacePipeline(pipeline=pipe)
-
-template = """Use the following pieces of context to answer the question at the end. If you don't know the answer,\
-just say that you don't know, don't try to make up an answer.
-
-Context: Large Language Models (LLMs) are the latest models used in NLP.
-Their superior performance over smaller models has made them incredibly
-useful for developers building NLP enabled applications. These models
-can be accessed via Hugging Face's `transformers` library, via OpenAI
-using the `openai` library, and via Cohere using the `cohere` library.
-
-Question: {question}
-Answer: """
-
-prompt = PromptTemplate(input_variables=["question"], template=template)
-llm_chain = LLMChain(prompt=prompt, llm=llm)
-
-# Use LangChain object
-question = "Which libraries and model providers offer LLMs?"
-response = llm_chain(prompt.format(question=question))
-print(f"Question: {question}")
-print(f"Response: {response['text']}")
+To run Q&A example with LangChain use script `run_pipeline_langchain.py` it supports similar syntax to `run_pipeline.py`. For example you can use following command.
```

Comment:
```suggestion
To run a Q&A example with LangChain, use the script `run_pipeline_langchain.py`. It supports a similar syntax to `run_pipeline.py`. For example, you can use following command:
```

Reply:
thanks, I did this change in latest commit, please have a look